e2e/node/server/express.ts

```diff
@@ -37,8 +42,19 @@ export function createApp(
 ) {
   const app = express();
 
-  // Initialised when the server comes up and is running...
-  let session: Session;
+  const sessionTokenSets = new Map<string, SessionTokenSet>();
+
+  app.use(
```

Comment:
The express app now gets the session from storage which will support additional tests when the API changes are made.